SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -681,16 +681,17 @@ ASpatialFunctionalTestFlowController* ASpatialFunctionalTest::GetFlowController(
 			return FlowController;
 		}
 	}
+	checkf(false, TEXT("We should always find a flow controller."));
 	return nullptr;
 }
 
 APlayerController* ASpatialFunctionalTest::GetFlowPlayerController(const ESpatialFunctionalTestWorkerType WorkerType, const int WorkerId)
```

Comment:
Could this return a reference at this point?

Reply:
ue4 gets upset when i try doing this `Missing '*' in Expected a pointer type`, something about it being a BlueprintPure ufunction I think

Reply:
Fair enough.